proxy/src/coco/source.rs

```diff
@@ -42,8 +42,27 @@ pub struct Person {
     pub avatar: String,
 }
 
-/// Representation of a code commit.
+/// Commit statistics.
+#[derive(Serialize)]
+pub struct CommitStats {
+    /// Additions.
+    pub additions: u64,
+    /// Deletions.
+    pub deletions: u64,
```

Comment:
`u64` ought to be enough for anybody. 😬

Reply:
You could use `usize` and avoid disabling the integer arithmetic and its warning of overflow.